docs/sources/operations/automatic-stream-sharding.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Automatic Stream Sharding
+menuTitle: Automatic stream sharding
+description: Automatic stream sharding can control issues around the per-stream rate limit
+weight: 110
+---
+
+# Automatic stream sharding
+
+Automatic stream sharding will attempt to keep streams under a `desired_rate` by adding new labels and values to
+existing streams. When properly tuned, this should eliminate issues where log producers are rate limited due to the
+per-stream rate limit.
+
+## The issue mitigated by automatic stream sharding
+
+Large log streams present several problems for Loki, namely increased and uneven resource usage on Ingesters and
+Distributors. The general recommendation is to explore existing log streams for additional label values that are both
+useful for querying and sufficiently low cardinality. There are many cases, however, where no more labels can
+be extracted, or cardinality for a label is dangerously large. To protect itself from such volume leading to operational failure, Loki implements per-stream rate limits;
+but the result is that some data is lost. The per-stream limit also needs human intervention to change, which is not ideal when log volumes increase and decrease.
+
+Automatic stream sharding is used to avoid large streams and rate limiting for any log stream by ensuring it is close to
```

Comment:
Passive voice (is used by who or what?).  Possible revisions to make this active voice: "Loki uses automatic stream sharding to avoid large streams...."  Or "You can use automatic stream sharding to avoid large streams...."